### PR TITLE
Fix for D3P_OCTOPUS_Driver2208_UART_BLTouch_LCD_BTT_TFT Build

### PR DESCRIPTION
**Pull request by andrewseago**
Fixed typo in Configuration_Tenlog.h that had falsesirius fm instead of false. 

Removed Y_Min = -5 changes for OpticalY. 

Modified Marlin/src/pins/stm32f4/pins_BTT_OCTOPUS_V1_common.h to better match more recent versions of the file in order to resolve build issues with D3P_OCTOPUS_Driver2208_UART_BLTouch_LCD_BTT_TFT

### CODE REVIEW

File: Marlin/Configuration_tenlog.h [hunk 1]
```diff
@@ -218,6 +218,7 @@
   #undef  Y_MIN_ENDSTOP_INVERTING
   #define Y_MIN_ENDSTOP_INVERTING true
   #define LUX_ENDSTOP_Y_OFFSET -8
+  #define LUX_REFERENCE_Y_MIN_POS -5
```
Comment (klack) [inline Marlin/Configuration_tenlog.h:221]: Looks like it is still in here

File: Marlin/Configuration_tenlog.h [hunk 2]
```diff
@@ -360,16 +361,17 @@
   #define E0_DRIVER_TYPE DriverType
   #define E1_DRIVER_TYPE DriverType
 #endif
-
-#define LUX_REFERENCE_Y_MIN_POS 0
+#ifndef LUX_REFERENCE_Y_MIN_POS
```
Comment (klack) [inline Marlin/Configuration_tenlog.h:364]: <-

File: Marlin/Configuration_tenlog.h [hunk 3]
```diff
@@ -360,16 +361,17 @@
...
+#endif
 #define LUX_REFERENCE_X1_MIN_POS -50
 #define LUX_REFERENCE_X2_MAX_POS 360
 #define LUX_REFERENCE_EXTRUDER_WIDTH 15
 
 #define X_MIN_POS LUX_REFERENCE_X1_MIN_POS -(LUX_EXTRUDER_X_OFFSET)
 #define X_MAX_POS LUX_MEASURED_X_BED_SIZE - LUX_REFERENCE_EXTRUDER_WIDTH
 
-#define Y_MIN_POS 0 //Always 0
+#define Y_MIN_POS LUX_REFERENCE_Y_MIN_POS //Always 0
```
Comment (klack) [inline Marlin/Configuration_tenlog.h:374]: <--

**Review by klack — CHANGES_REQUESTED** [review]

**Comment by klack** [comment]
Nice description

**Comment by andrewseago** [comment]
Requested changes should be applied now.

File: Marlin/Configuration_tenlog.h [hunk 4]
```diff
@@ -369,7 +368,7 @@
...
```
Comment (klack) [inline Marlin/Configuration_tenlog.h:371]: On last thing, can you put this back to 0

**Review by klack — CHANGES_REQUESTED** [review]

**Pull request closed without merging**